packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -457,3 +457,116 @@ def principal():
                 'unreserve_resources': resources}
             r = dcos_api_session.post('/mesos/api/v1', json=request)
             assert r.status_code == 202, r.text
+
+
+def reactivate_agent(dcos_api_session, agent_id):
```

Comment:
This is invoked from test teardowns and we should make sure that it succeeds. We should probably add retry logic to it either globally or on a per-invocation basis (for all cleanups).